style.css

```diff
@@ -0,0 +1,30 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&family=Quicksand:wght@300;400&display=swap');
+body{
+    background-color: rgba(0, 0, 0, .9);
+    font-family: 'Inter', sans-serif;
+}
+.name{
+    margin-bottom: 0px;
+    font-size: 40px;
+    font-weight: 700;
+}
+.aboutMe{
+    color: #fff;
+    max-width: 530px;
+    margin-left: 300px;
+    margin-top: 300px;
+}
+@media screen and (min-width: 400px) {
```

Comment:
Media Queries should be placed at the bottom of a css file.

Css reads from top to bottom this will hep with debugging issues in the future